force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -355,15 +449,16 @@ public with sharing class ClassesObjectsMethods {
      * @return A list of new Person objects.
      */
     public static List<Person> createMultiplePersons(List<String> names, Integer age) {
-
+        List<Person> personList = new List<Person>();
         // Loop through each name in the provided list of names
+        for (String name: names) {
             // Create a new Person object for each name
-            
+            Person newPerson = new Person(name, age);
             // Add the new Person object to the list of Person objects
-        
-        
+            personList.add(newPerson); 
+        }         
         // Return the list of Person objects
-        return null; // Replace null with the variable you used to store the result
+        return personList; // Replace null with the variable you used to store the result
```

Comment:
Good work. If you want you can add that value directly to the list in your loop and skip the variable. 
```
 public static List<Person> createMultiplePersons(List<String> names, Integer age) {
        List<Person> persons = new List<Person>();
        for (String name : names) {
            persons.add(new Person(name, age));
        }
        return persons;
    }
```